src/future/mod.rs

```diff
@@ -47,8 +47,11 @@
 //! [`Future::try_race`]: trait.Future.html#method.try_race
 
 cfg_alloc! {
+    pub use into_future::IntoFuture;
```

Comment:
If you plan to replace this with re-export of std IntoFuture when std IntoFuture's is stable, that would be a breaking change. Theoretically, there is a way to avoid almost breakage. See https://github.com/rust-lang/futures-rs/issues/2207 and https://github.com/taiki-e/futures-compat-experiment for that.